<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewGoodsDialog</class>
 <widget class="QDialog" name="ViewGoodsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>12</y>
     <width>521</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="yearBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="monthBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="dayBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="queryGoodsButton">
        <property name="text">
         <string>查询货品</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="goodsView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
